.github/workflows/build-on-commit.yml

```
@@ -0,0 +1,98 @@
+name: Build Commit
+on: [push]
+jobs:
+  build-elmu:
+    runs-on: ubuntu-latest
+    env:
+      NODE_ENV: development
+      ELMU_ENV: test
+    services:
+      minio:
+        image: bitnami/minio:2020.12.18
+        ports:
+          - 9000:9000
+        env:
+          MINIO_ACCESS_KEY: "UVDXF41PYEAX0PXD8826"
+          MINIO_SECRET_KEY: "SXtajmM3uahrQ1ALECh3Z3iKT76s2s5GBJlbQMZx"
+          MINIO_BROWSER: "on"
+          MINIO_DOMAIN: "localhost"
+      mongo:
+        image: mongo:4.2.11-bionic
+        ports:
+          - 27017:27017
+      maildev:
+        image: maildev/maildev:1.1.0
+        ports:
+          - 8000:8000
+    steps:
+      - name: Change user
+        run: sudo chown -R $USER:$USER ${{ github.workspace }}
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "16"
+      - uses: actions/cache@v2
+        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+        with:
+          path: ~/.cache/yarn
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+      - name: Install node modules
+        run: yarn install --non-interactive --check-files --frozen-lockfile
+      - name: Prepare build
+        run: ./node_modules/.bin/gulp ciPrepare || echo "Failed"
```

We have to remove the `||`, otherwise it will silently continue on errors